webapp/packages/core-blocks/src/AppRefreshButton.tsx

```diff
@@ -6,47 +6,28 @@
  * you may not use this file except in compliance with the License.
  */
 import type React from 'react';
-import styled, { css } from 'reshadow';
 
 import { App, useService } from '@cloudbeaver/core-di';
 
-const style = css`
-  button {
-    color: white;
-    font-weight: 500;
-    background-color: #2a7cb4;
-    padding: 5px 16px;
-    border-radius: 4px;
-    letter-spacing: 0.08929em;
-    font-size: 0.875rem;
-    text-transform: uppercase;
-    cursor: pointer;
-    outline: none;
-    border: none;
-    &:hover,
-    &:focus {
-      opacity: 0.8;
-    }
-    &:active {
-      opacity: 0.5;
-    }
-  }
-`;
+import style from './AppRefreshButton.m.css';
+import { s } from './s';
+import { useS } from './useS';
 
 interface IProps {
   className?: string;
 }
 
 export const AppRefreshButton: React.FC<IProps> = function AppRefreshButton({ className }) {
+  const styles = useS(style);
```

Comment:
please don't use `useS` here because this component can appear on the wrong app initialization screen